style_guide/README.md

```diff
@@ -0,0 +1,307 @@
+are already completed.
+
+### Nominal releases
+
+1. Make sure the `dev` branch is up-to-date with any changes you want included in the release (i.e. merge in any feature
+   branches using the nominal [git/GitHub workflow](#git-&amp-github-workflow)).
+2. Create a new 'release' branch off of `dev`.  The name of the branch should match the version number to be used for
+   the release, which should follow the [versioning](#versioning) conventions.
+3. Make any release-specific commits to the new release branch using the nominal `git add`/`git commit` cycle. This may
+   include commits that add release notes, or update version numbers in various configurations.
+4. Push the release branch to the main `IMAP-Science-Operations-Center` `imap_processing` repo (i.e. `upstream`).
+5. In GitHub, create two pull requests: one that merges the release branch into `main`, and one that merges the release
+   branch into `dev`. Proceed with the nominal review & merge process described in steps (10) and (11) in the 
+6. proceed with the nominal review & merge process described in steps (10) and (11) in the
+   [git/GitHub workflow](#git-&amp-github-workflow) section.
+7. Once the changes are merged into the `main` branch, create a
+   [new release](https://github.com/IMAP-Science-Operations-Center/imap_processing/releases) for the merge commit and
+   assign the appropriate version number.
+
+### Making hotfixes
+
+1. Create a new branch named `hotfix-<description>` off of the `main` branch, and commit any necessary changes following
+   the nominal `git add`/`git commit` cycle.
+2. Push the hotfix branch to the main `IMAP-Science-Operations-Center` `imap_processing` repo (i.e. `upstream`), and
+   open two separate pull requests: one that merges the hotfix branch into the `main` branch, and one that merges the
+   hotfix branch into the `dev` branch.
+3. For each of these pull requests, proceed with the nominal review & merge process described in steps (10) and (11) in
+   the previous section.
+4. Once the changes are merged into the `main` branch, create a
+   [new release](https://github.com/IMAP-Science-Operations-Center/imap_processing/releases) or a
+   [new tag](https://github.com/IMAP-Science-Operations-Center/imap_processing/tags) for the merge commit and assign the
+   appropriate [version number](#versioning).
+
+
+## Checklist for Contributors and Reviewers of Pull Requests
+
+The following is a guide to be used for contributors and reviewers of `imap_processing` pull requests. Note that this is
+only a guide; it should not be treated as a fully comprehensive, foolproof list that must be used in all situations, and
+parts of it are subjective.
+
+If the contributor/reviewer can answer "yes" to all the following questions, then conceivably the proposed changes are
+acceptable and the PR can be reviewed and merged.
+
+### Checklist for Contributors
```

Comment:
Is Contributors both reviewers and the person who is creating the PR?

Reply:
My intention is that the 'contributor' is the person who is creating the PR.  The 'reviewer' is whoever that person assigned as a reviewer.